### PR TITLE
AppTP Onboarding, FAQ, Copy Updates

### PR DESCRIPTION
**Pull request by SlayterDev**
Task/Issue URL: https://app.asana.com/0/715106103902962/1204714133207123/f
Tech Design URL:
CC: @samsymons 

**Description**:
This PR introduces the onboarding flow for AppTP as well as the AppTP screen and beta copy updates.

**Steps to test this PR**:
1. On a fresh install tapping AppTP in settings should take you through the onboarding flow
2. The final page should prompt for VPN permissions and enable the VPN
3. The FAQ button in the activity view and the "Learn More" button in onboarding should open the FAQ page
4. The "What Are App Trackers" button in the activity view should open said article
5. Refer to asana task to verify copy changes

**Copy Testing**:

* [x] Use of correct apostrophes in new copy, ie `’` rather than `'`

**Orientation Testing**:

* [x] Portrait
* [x] Landscape

**Device Testing**:

* [ ] iPhone SE (1st Gen)
* [ ] iPhone 8
* [ ] iPhone X
* [x] iPhone 14 Pro
* [x] iPad

**OS Testing**:

* [ ] iOS 14
* [ ] iOS 15
* [x] iOS 16

**Theme Testing**:

* [x] Light theme
* [x] Dark theme

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

File: DuckDuckGo/AppTPAboutView.swift
Comment (brindy) [inline DuckDuckGo/AppTPAboutView.swift]: typo here (Backgroud -> Background)

File: DuckDuckGo/AppTPAboutView.swift
Comment (brindy) [inline DuckDuckGo/AppTPAboutView.swift]: doesn't seem any benefit to this? might as well inline it

File: DuckDuckGo/AppTPFAQView.swift
Comment (brindy) [inline DuckDuckGo/AppTPFAQView.swift:78]: Where did this colour come from?  We should only be using Design System colours now.

File: DuckDuckGo/AppTPFAQView.swift
Comment (brindy) [inline DuckDuckGo/AppTPFAQView.swift]: Fonts should all come from the Design System now.

File: DuckDuckGo/OnboardingContainerView.swift
Comment (brindy) [inline DuckDuckGo/OnboardingContainerView.swift]: As other comments, these should come from design system.  You can `import DesignResourcesKit` and see what's available in there.  If there's stuff in Figma that isn't in there, please push back on the designer.

File: DuckDuckGo/OnboardingContainerView.swift
Comment (brindy) [inline DuckDuckGo/OnboardingContainerView.swift]: Design System

File: DuckDuckGo/OnboardingStepView.swift
Comment (brindy) [inline DuckDuckGo/OnboardingStepView.swift]: DesignSystem :)

File: DuckDuckGo/SettingsViewController.swift
Comment (brindy) [inline DuckDuckGo/SettingsViewController.swift]: Can you use a consistent name here, please?  I don't mind which

File: DuckDuckGo/AppTPFAQView.swift
Comment (brindy) [reply DuckDuckGo/AppTPFAQView.swift]: In figma you can see what font it should be by clicking into the text and it'll tell the Typography name.  Then in `DesignResourcesKit` there should be a corresponding font already defined.  If figma contains something not in the design system please push back on to the designer. 

Screenshot 2023-06-30 at 13 37 44

See https://github.com/duckduckgo/DesignResourcesKit/blob/main/Sources/DesignResourcesKit/Font.swift

**Review by brindy — DISMISSED** [review]
I haven't tested this, I'll leave that to whoever you identified in your kick off (@samsymons  I presume?), but I was curious what you were up to after your question on MM. 

I've left a bunch of comments related to the design system (and couple of other things).  Please ping on MM if you need any help!

File: DuckDuckGo/AppDelegate.swift
Comment (samsymons) [inline DuckDuckGo/AppDelegate.swift]: Adding a note to remove this before merging

File: DuckDuckGo/AppTPAboutView.swift
Comment (samsymons) [inline DuckDuckGo/AppTPAboutView.swift]: Since this `Group` is technically just containing a single view (three `Text` views concatenated), is it redundant? I tested it just now in a preview and the view appeared the same after removing the group.

File: DuckDuckGo/AppTPAboutView.swift
Comment (samsymons) [inline DuckDuckGo/AppTPAboutView.swift]: Should hopefully be able to use DesignResourcesKit for these fonts

File: DuckDuckGo/UserText.swift
Comment (samsymons) [inline DuckDuckGo/UserText.swift:589]: Is it deliberate that we're truncating this?

**Review by samsymons — APPROVED** [review]
New DRK usage looks good, thanks! :+1: Onboarding changes look good overall.

We may need to translate this PR as well, even though this feature is internal. Feel free to merge anyway, and we can translate after the fact.

File: DuckDuckGo/UserText.swift
Comment (SlayterDev) [reply DuckDuckGo/UserText.swift:589]: Yes this was a copy change that was requested

**Pull request merged**
